JitsiConference.js

```diff
@@ -964,6 +964,16 @@ JitsiConference.prototype._fireMuteChangeEvent = function(track) {
         actorParticipant = this.participants[actorId];
     }
 
+    // Setup E2EE on the sender that is created for the unmuted track.
+    if (this._e2eeCtx && !track.isMuted() && browser.doesVideoMuteByStreamRemove()) {
```

Comment:
all those little ifs here and there may finally get out of control at some point, and all end up in the JitsiConference which is now 3k lines long

Reply:
what I mean by that is that it may be better off having e2ee logics in a separate module that works with events coming from other modules

Reply:
Makes sense. I'll keep that in mind while working on adding Olm, which will only add more stuff to this. We probably need new events, but that's ok.